README.md

```diff
@@ -53,20 +53,20 @@ $: bitbake core-image-minimal
 # Additional Information
 
 ## Optimize build time:
-Add [DL_DIR](https://docs.yoctoproject.org/ref-manual/variables.html#term-DL_DIR) and [SSTATE_DIR](https://docs.yoctoproject.org/ref-manual/variables.html#term-SSTATE_DIR) configs in conf/local.conf with respective PATH variable.
+Add [DL_DIR](https://docs.yoctoproject.org/ref-manual/variables.html#term-DL_DIR) and [SSTATE_DIR](https://docs.yoctoproject.org/ref-manual/variables.html#term-SSTATE_DIR) configs to conf/local.conf with a respective PATH variable.
```

Comment:
Update this "configs" to "configurations" as well

Reply:
Thanks for point out, Corrected.